### ISSUE TITLE
Result grid not emptying when full search returns no items

### ISSUE THREAD

**Issue by emilioidk** [memefulsearch/memefulsearch.github.io#2]
You might want to empty the result grid and return a failure message when the search term returns no items. Perhaps this would be a bad feature for terms that yielded results a character or two before being completed, but for long terms, it makes no sense.

Example:
- "adventure timess" doesn't yield any result, but makes sense to return the results from "adventure time" that has one character less and does return gifs
- "argentina" doesn't yield any results, and the last substring from initial character that does return results is "arge"...most certainly the results don't belong to the search term.

**Comment by memefulsearch** [comment]
@emilioidk Yes that sounds like a good improvement, I'll look into that at some point. Although, you can branch if you will, would happily merge it.

**Comment by ennalikewine** [comment]
@memefulsearch
Hi Do you know Tom? I'll leave it at that. 
How can I help with the growth of memefulsearch.github.io I have noticed, you've been adding the missing images. 
Are there specific ones you need. I don’t mind working collectively but I would prefer to work efficiently. I can do the r/d on the images but if I can skip a few that would definitely speed things up. 
Does that make sense? 
Thanks.
